### ISSUE TITLE
Wallet Accessibility Audit

### ISSUE THREAD

**Issue by kylehickinson** [brave/brave-ios#4303]
### Description: 

Wallet should be audited for accessibility before shipping:

- [ ] VoiceOver grouping, labels, values, etc.
- [ ] VoiceOver gestures
- [ ] Dynamic font sizes
- [ ] Color contrasts

**Comment by petergeorgas** [comment]
I would like to take a shot at this.

**Comment by LarrySOScary** [comment]
> ### Description: 
> 
> 
> 
> Wallet should be audited for accessibility before shipping:
> 
> 
> 
> - [ ] VoiceOver grouping, labels, values, etc.
> 
> - [ ] VoiceOver gestures
> 
> - [ ] Dynamic font sizes
> 
> - [ ] Color contrasts

If you aren't making fun of me then thank you, I need some help here for sure... for starters, are you talking about html accessibility here or are you just pulling my leg with a bunch of complex sounding yet (to  the budding code novice) cautiously plausible expert jargon?
